[2021-08-13 08:39:44,457][MainThread:8740][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 08:48:26,279][MainThread:8740][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 08:48:27,302][MainThread:11632][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 09:10:27,822][Thread-1:9012][task_id:lts.views][views.py:207][INFO][登录成功]
[2021-08-13 10:43:52,021][Thread-3:10376][task_id:django.request][log.py:230][ERROR][Internal Server Error: /logistic/LogisticOperationSystem]
Traceback (most recent call last):
  File "D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py", line 6355, in get
    drs = db.execute_sql(conn, sql)
  File "D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\common\db.py", line 45, in execute_sql
    cursor.execute(sql)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\pymysql\cursors.py", line 148, in execute
    result = self._query(query)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\pymysql\cursors.py", line 310, in _query
    conn.query(q)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\pymysql\connections.py", line 548, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\pymysql\connections.py", line 775, in _read_query_result
    result.read()
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\pymysql\connections.py", line 1156, in read
    first_packet = self.connection._read_packet()
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\pymysql\connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\pymysql\protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\pymysql\err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1046, 'No database selected')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\django\views\generic\base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\django\views\generic\base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py", line 6397, in get
    result = json.dumps(result)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\json\__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\json\encoder.py", line 180, in default
    o.__class__.__name__)
TypeError: Object of type 'OperationalError' is not JSON serializable
[2021-08-13 11:45:13,595][Thread-4:20400][task_id:lts.views][views.py:298][INFO][登录成功]
[2021-08-13 11:48:08,486][Thread-5:5444][task_id:lts.views][views.py:207][INFO][登录成功]
[2021-08-13 11:53:15,082][Thread-11:21900][task_id:lts.views][views.py:298][INFO][登录成功]
[2021-08-13 11:54:58,699][Thread-12:13708][task_id:lts.views][views.py:298][INFO][登录成功]
[2021-08-13 13:09:05,348][Thread-16:17736][task_id:django.request][log.py:230][ERROR][Internal Server Error: /logistic/LogisticOperationSystem]
Traceback (most recent call last):
  File "D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py", line 6355, in get
    drs = db.execute_sql(conn, sql)
  File "D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\common\db.py", line 45, in execute_sql
    cursor.execute(sql)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\pymysql\cursors.py", line 148, in execute
    result = self._query(query)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\pymysql\cursors.py", line 310, in _query
    conn.query(q)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\pymysql\connections.py", line 548, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\pymysql\connections.py", line 775, in _read_query_result
    result.read()
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\pymysql\connections.py", line 1156, in read
    first_packet = self.connection._read_packet()
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\pymysql\connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\pymysql\protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\pymysql\err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1046, 'No database selected')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\django\views\generic\base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\site-packages\django\views\generic\base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py", line 6397, in get
    result = json.dumps(result)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\json\__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\zhi.zhou\Anaconda3\envs\lts\lib\json\encoder.py", line 180, in default
    o.__class__.__name__)
TypeError: Object of type 'OperationalError' is not JSON serializable
[2021-08-13 13:09:13,578][Thread-16:17736][task_id:lts.views][views.py:298][INFO][登录成功]
[2021-08-13 13:32:14,848][MainThread:8832][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 14:03:48,583][Thread-1:18364][task_id:lts.views][views.py:207][INFO][登录成功]
[2021-08-13 14:55:01,072][MainThread:8832][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 14:55:03,095][MainThread:24240][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 14:56:49,358][MainThread:24240][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 14:56:50,418][MainThread:25480][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 15:03:10,437][MainThread:25480][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 15:03:11,546][MainThread:25420][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 15:10:30,677][MainThread:25420][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 15:10:32,296][MainThread:27048][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 15:24:23,085][MainThread:27048][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 15:24:24,520][MainThread:23908][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 15:27:19,907][MainThread:23908][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 15:27:20,965][MainThread:10524][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 15:27:30,236][MainThread:10524][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 15:27:31,319][MainThread:26784][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 15:27:45,580][MainThread:26784][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 15:27:46,651][MainThread:25160][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 15:38:40,887][MainThread:25160][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 15:38:42,479][MainThread:21296][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 15:41:22,737][MainThread:21296][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 15:41:23,803][MainThread:10804][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 15:43:39,059][MainThread:10804][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 15:43:40,148][MainThread:21412][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 15:44:09,524][Thread-1:8784][task_id:lts.views][views.py:207][INFO][登录成功]
[2021-08-13 15:47:57,331][MainThread:21412][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 15:47:58,719][MainThread:27356][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 15:49:07,135][MainThread:27356][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 15:49:08,262][MainThread:26764][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 15:50:05,459][MainThread:26764][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 15:50:06,544][MainThread:21064][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 15:52:46,177][Thread-2:25260][task_id:django.request][log.py:230][WARNING][Not Found: /favicon.ico]
[2021-08-13 15:53:58,018][MainThread:21064][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 15:53:59,174][MainThread:21884][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 15:54:33,451][MainThread:21884][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 15:54:34,520][MainThread:21852][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 15:55:33,921][MainThread:21852][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 15:55:34,999][MainThread:24144][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 15:55:37,780][Thread-1:20060][task_id:django.request][log.py:230][WARNING][Not Found: /favicon.ico]
[2021-08-13 15:57:25,117][MainThread:24144][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 15:57:26,534][MainThread:25424][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 15:57:54,511][MainThread:25424][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 15:57:56,324][MainThread:23016][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 15:58:51,171][MainThread:23016][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 15:58:52,528][MainThread:1800][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 16:01:27,806][MainThread:1800][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 16:01:29,302][MainThread:17492][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 16:03:19,932][MainThread:17492][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 16:03:21,717][MainThread:25112][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 16:04:38,752][MainThread:25112][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 16:04:39,942][MainThread:26448][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 16:05:52,211][MainThread:26448][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 16:05:53,282][MainThread:19560][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 16:08:17,333][MainThread:19560][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 16:08:18,457][MainThread:20424][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 16:08:40,960][MainThread:20424][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 16:08:42,157][MainThread:25156][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 16:10:42,376][Thread-2:27256][task_id:lts.views][views.py:207][INFO][登录成功]
[2021-08-13 16:11:20,985][MainThread:25156][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 16:11:22,100][MainThread:16352][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 16:13:11,116][MainThread:16352][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 16:13:12,169][MainThread:22908][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 16:14:58,839][MainThread:22908][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 16:14:59,987][MainThread:22212][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 16:19:08,412][Thread-1:19672][task_id:lts.views][views.py:207][INFO][登录成功]
[2021-08-13 16:30:09,774][Thread-4:25660][task_id:lts.views][views.py:207][INFO][登录成功]
[2021-08-13 16:41:51,555][Thread-8:27312][task_id:django.request][log.py:230][WARNING][Not Found: /logistic/LogisticNoPageProductInfoSearch]
[2021-08-13 16:42:06,742][Thread-9:20564][task_id:lts.views][views.py:298][INFO][登录成功]
[2021-08-13 16:49:29,298][MainThread:22212][task_id:django.utils.autoreload][autoreload.py:251][INFO][D:\zhouzhi\gitlit_code\logistic_tracking_system\logistic_tracking_system\lts\views.py changed, reloading.]
[2021-08-13 16:49:30,666][MainThread:27200][task_id:django.utils.autoreload][autoreload.py:637][INFO][Watching for file changes with StatReloader]
[2021-08-13 16:58:32,228][Thread-4:11764][task_id:django.request][log.py:230][WARNING][Not Found: /logistic/LogisticNoPageProductInfoSearch]
